Modules/Server/Sources/PocketCastsServer/Public/StatsManager.swift

```diff
@@ -29,6 +29,39 @@ public class StatsManager {
         }
     }
 
+    func updateStatsIfNeeded(savedDynamicSpeed: TimeInterval, savedVariableSpeed: TimeInterval, totalListenedTo: TimeInterval, totalSkipped: TimeInterval, savedAutoSkipping: TimeInterval) {
+        let minimumStatsChangeToUpdate: TimeInterval = 100
+
+        updateQueue.sync {
+            if savedDynamicSpeed - self.savedDynamicSpeed > minimumStatsChangeToUpdate {
```

Comment:
Nit: Do you want to save only if the difference is positive right? No need to use `abs` ?